sl/symbin.cc

```diff
@@ -1213,14 +1218,17 @@ bool BuiltInTable::handleBuiltIn(
 
     TMap::const_iterator it = tbl_.find(name);
     if (tbl_.end() == it) {
-        static const char namePrefixNondet[] = "__VERIFIER_nondet_";
+        static const char namePrefixNondet[] = "__VERIFIER_nondet";
+        static const char namePrefixObjSize[] = "llvm.objectsize.i";
```

Comment:
This only works because `sizeof(namePrefixNondet) == sizeof(namePrefixObjSize)`.  Please put there a static assert on the size and a comment stating the assumption.